manifests/user.pp

```diff
@@ -114,6 +115,8 @@
     require => User[$user]
   }
 
+  File <<| tag == $::fqdn |>>
```

Comment:
Collecting all file exported resources based on an fqdn tag seems risky to me - can we collect in a more controlled way? This introduces huge cross-environment potential for problems.